ejabberd.te

```diff
@@ -0,0 +1,81 @@
+policy_module(ejabberd,0.0)
+
+
+gen_require(`
+    type net_conf_t;
+    class lnk_file read;
+    class tcp_socket { accept bind connect create getattr getopt listen read setopt write };
+    class udp_socket { bind connect create getattr getopt read setopt write };
+    class unix_dgram_socket { connect create getopt setopt write };
+')
+
+
+# Private type declarations
+type ejabberd_t;
+type ejabberd_exec_t;
+type ejabberd_var_lib_t;
+type ejabberd_var_log_t;
+
+files_type(ejabberd_var_lib_t)
+
+init_daemon_domain(ejabberd_t, ejabberd_exec_t)
+init_daemon_domain(ejabberd_exec_t, init_t)
```

Comment:
this call does not make any sense, ```init_daemon_domain``` expects a domain and a exec_file type for a transition from init_t over the exec_file type to the specified domain